RLTSquare/CustomEmailSender/etc/email_templates.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Email:etc/email_templates.xsd">
+    <template id="test_email_template" label="This is test email" file="test_email_template.html" type="html"
```

Comment:
Naming convention - the module is named "CustomEmailSender", so it would be nice if the identifiers, names, etc in xml files followed the same naming pattern...